src/scripts/sync_synonym.py

```diff
@@ -0,0 +1,90 @@
+"""Whenever new synonym or label in upstream source, want to add it to Mondo. If the label is not the Mondo label or
+existing synonym, it will become a synonym.
+"""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Union, List, Tuple, Dict
+
+import pandas as pd
+from oaklib import get_adapter
+from oaklib.implementations import SqlImplementation
+from oaklib.types import CURIE, PRED_CURIE
+
+
+# todo: reactivate if needed
+# HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+# SRC_DIR = HERE.parent
+# PROJECT_ROOT = SRC_DIR.parent
+# sys.path.insert(0, str(PROJECT_ROOT))
+# from src.scripts.utils import ONTOLOGY_DIR
+
+
+
+# TODO
+def sync_synonyms(
+    ontology_db_path: Union[Path, str], mondo_db_path: Union[Path, str], mondo_mappings_path: Union[Path, str],
+    onto_config_path: Union[Path, str], outpath_added: Union[Path, str], outpath_updated: Union[Path, str]
+):
+    """Whenever new synonym or label in upstream source, want to add it to Mondo. If the label is not the Mondo label or
+     existing synonym, it will become a synonym."""
+    #     For every term in source:
+    #         For every synonym or label on term:
+    #             Add row to robot template with correct evidence (with Mondo ID of course)
+    source_db: SqlImplementation = get_adapter(ontology_db_path)
+    source_ids_all: List[CURIE] = [
+        x for x in [y for y in source_db.entities(filter_obsoletes=False)]]
+    # if any([x.startswith(y) for y in owned_prefix_map.keys()]  # todo: filter by owned_prefix_map in metadata
+    id_syns_map: Dict[CURIE, List[Tuple[PRED_CURIE, str]]] = {_id: [] for _id in source_ids_all}
+    for source_id in source_ids_all:
+        for pred, alias in sorted(source_db.alias_relationships(source_id)):
```

Comment:
## FYI: Synonym querying updates
@twhetzel Just wanted to update you how far I got today. It is indeed pretty easy to get these synonyms using OAK. Here's an example of what comes out for a given term:

```py
>>> pprint(id_syns_map2['OMIM:100100'])
[('oio:hasExactSynonym', 'PBS'),
 ('oio:hasExactSynonym',
  'abdominal muscles, absence of, with urinary tract abnormality and '
  'cryptorchidism'),
 ('oio:hasExactSynonym', 'eagle-barrett syndrome'),
 ('oio:hasExactSynonym', 'prune belly syndrome'),
 ('rdfs:label', 'prune belly syndrome')]
```

My thought on next step following this point in the code:
1. Get exact matched Mondo IDs using `mondo.sssom.tsv`
2. Query `mondo.db` for current synonym info on that term